CentOS-7/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN yum -y update; yum -y install \
     glibc-devel.x86_64 \
     gmp-devel.x86_64 \
     libstdc++-devel.x86_64 \
+    intel-tbb \
```

Comment:
It should be `tbb-devel`, like on Fedora, because CentOS is derived from older versions of Fedora.